.github/workflows/phpunit-tests.yml

```diff
@@ -39,9 +39,15 @@ jobs:
                     - '8.1'
                     - '8.2'
                 multisite: [false, true]
+                wordpress: [''] # Latest WordPress version.
+                include:
+                    # Test with the previous WP version.
+                    - php: '7.4'
+                      wordpress: '6.0'
 
         env:
             WP_ENV_PHP_VERSION: ${{ matrix.php }}
+            WP_ENV_CORE: WordPress/WordPress${{ matrix.wordpress != '' && format( '#{0}-branch', matrix.wordpress ) || '' }}
```

Comment:
The advantage of using the `WordPress/WordPress` git repo's `x.y-branch` is that we don't have to compute the latest .patch version in that series (would be 6.0.3 here).